validate-microshift/kuttl-test.sh

```diff
@@ -5,14 +5,30 @@ set -x
 
 ROOT="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")/../")"
 
-KUTTL_VERSION="0.10.0"
+KUTTL_VERSION="0.15.0"
 KUTTL="$ROOT/bin/kuttl"
```

Comment:
While we're updating this script, could we change this to put the file in the build output directory that is already ignored by git? That makes it nicer when running the script on a dev system.

```suggestion
KUTTL="$ROOT/_output/bin/kuttl"
```